cms/templates/partials/admissions-section.html

```diff
@@ -0,0 +1,43 @@
+{% load image_version_url wagtailcore_tags %}
+<div class="container-fluid mb-4 admissions-section">
+    <div class="row">
+        <div class="container admissions-section">
+            <div class="row">
+                <div class="col-md-5 col-sm-12 mb-2">
+                    <h1 class="section-heading">Information</h1>
+                    <h2 class="admissions-heading mt-2">Admissions</h2>
+                    <a href="/admissions" class="btn btn-primary btn-apply mt-4 mb-2">Apply Now</a>
+                    {% if page.notes %}
+                        <br />
+                        <small class="ml-3">*{{ page.notes }}</small>
+                    {% endif %}
+                </div>
+                <div class="col-md-7 col-sm-12 border-left border-dark">
+                    {{ page.details|richtext }}
+                    <img src="{% image_version_url page.workflow_infographics "max-600x400" %}" width="100%" height="auto" alt="bootcamp steps">
```

Comment:
either marked the `workflow_infographics` image required in CMS or check for it availability before rendering, else there will be an `AttributeError`.

Reply:
Template tags do not raise `AttributeError` but that's fine, will add a check.

Reply:
anyway, I am getting `AttributeError`.

Reply:
Updated anyway